sonar-project.properties

```diff
@@ -1,3 +1,5 @@
 sonar.projectKey=SonarSource_docker-sonarqube_AYcnOvlJTpBOcQuGEdI5
-sonar.sources=9/,commercial-editions/,community-build/
+sonar.sources=9/,commercial-editions/,community-build/,docker-official-images/
 sonar.organization=sonarsource
+sonar.go.coverage.reportPaths=docker-official-images/coverage.out
+sonar.exclusions=docker-official-images/**/*_test.go
```

Comment:
Why excluding the tests from the analysis?

Reply:
right, i used the wrong property, there is a dedicated sonar.tests